test/extensions/filters/http/cache/cache_filter_integration_test.cc

```diff
@@ -286,9 +361,11 @@ TEST_P(CacheIntegrationTest, ServeHeadRequest) {
   {
     // Since it is a head request, no need to encodeData => the response_body is absl::nullopt.
     IntegrationStreamDecoderPtr response_decoder = sendHeaderOnlyRequestAwaitResponse(
-        request_headers,
-        simulateUpstreamResponse(response_headers,
-                                 /*response_body*/ makeOptRefFromPtr<const std::string>(nullptr)));
+        request_headers, simulateUpstreamResponse(
+                             response_headers,
+                             /*response_body*/ makeOptRefFromPtr<const std::string>(nullptr),
```

Comment:
nit: this pattern was pre-existing in this file, but you can make an empty optref with:
```
OptRef<const std::string>()
```
here and below.

You could also just declare
```
  OptRef<const std::string> empty_body_;
  OptRef<const Http::TestResponseTrailerMapImp> empty_trailers_;
```
in the test class and pass that in, to avoid the `/*response_body=*/` syntax, just to make the call-sites more concise.

Reply:
Uuu, this is a great suggestion! Thank you!